### DISCUSSION TITLE
how do you enable response data fields in requests

### DISCUSSION THREAD

**Discussion by zhenisduissekov** [slanatech/swagger-stats#252] [Q&A]
swagger-stats Last Errors and Longest Errors are very useful for debugging. It enables me to see what fields were input by the user, but it would even more helpful if I could see the actual response from the system (its fields). Right now it shows that it is a successful response or not. But what actually was returned is not clear. How can I enable this field? Is it actually possible?

thank you 
your product is great!

**Comment by sv2** [comment]
Thank you ! Do you mean headers and content of the response ?

**Reply by zhenisduissekov** [reply]
yes the content of response would be so powerful for troubleshooting
